client/ayon_traypublisher/plugins/create/create_csv_ingest.py

```diff
@@ -770,17 +770,28 @@ def _create_instances_from_csv_data(self, csv_dir: str, filename: str):
                 ),
                 None
             )
+
             slate_exists: bool = any(
                 repre_item.slate_exists
                 for repre_item in repre_items
             )
 
+            is_reviewable: bool = any(
+                True
+                for repre_item in repre_items
+                if "review" in repre_item.tags
+            )
+
             families: List[str] = ["csv_ingest"]
             if slate_exists:
                 # adding slate to families mainly for loaders to be able
                 # to filter out slates
                 families.append("slate")
 
+            if is_reviewable:
+                # review family needs to be added for ExtractReview plugin
+                families.append("review")
```

Comment:
Doesn't this also require `instance.data["review"] = True` (I don't recall anymore where that was used?)

Reply:
so yeah it is